tools/obscuroscan_v3/frontend/src/components/modules/dashboard/recent-blocks.tsx

```diff
@@ -13,7 +13,7 @@ export function RecentBlocks({ blocks }: { blocks: any }) {
           </Avatar>
           <div className="ml-4 space-y-1">
             <p className="text-sm font-medium leading-none">
-              #{block?.blockHeader?.number}
+              #{+block?.blockHeader?.number}
```

Comment:
Is that `+` a typo or some javascript magic?

Reply:
It's same as using `{Number(block?.blockHeader?.number)}`